clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/operations/OperationCode.java

```diff
@@ -18,11 +18,15 @@
 
 import org.ehcache.clustered.client.internal.store.operations.codecs.OperationCodecFactory;
 
+import com.sun.org.apache.regexp.internal.RE;
+
 import static org.ehcache.clustered.client.internal.store.operations.codecs.OperationCodecFactory.PutOperationCodecFactory;
 
 public enum OperationCode {
 
-  PUT((byte)1, new PutOperationCodecFactory());
+  PUT((byte)1, new PutOperationCodecFactory()),
```

Comment:
Now that I look at this I wonder if there aren't too many unneeded abstraction layers here. I don't think we need a per-operation codec: one codec should suffice for all operations as the encoding / decoding is always only about the opcode, the key and the value. Having a single `OperationCodec` class would certainly simplify things a lot.